R/series__series.R

```diff
@@ -40,6 +45,33 @@
 #'   that we can use this information later on to avoid re-sorting it.
 #' - `SORTED_DESC` is similar but applies to sort in decreasing order.
 #'
+#' ## name
+#'
+#' `$name` returns the name of the Series.
+#'
+#' ## shape
+#'
+#' `$shape` returns a numeric vector of length two with the number of length
+#' of the Series and width of the Series (always 1).
+#'
+#' @section Sub-namespaces:
+#'
+#' Some functions are stored under active bindings, so it works like a sub-namespaces.
+#'
+#' ## expr
+#'
+#' `$expr` is a shorthand of writing something like
+#' `pl$select(s)$select(pl$col(<name>)$<Expr>)$to_series(0)`.
+#' This subnamespace is experimental.
```

Comment:
Just to be sure, this is a workaround because `pl$Series(c(1:3))$mean()` doesn't work for example, right? If so it would be useful to add a note that explains the diff with the python syntax (that just needs `.mean()`).

Reply:
`Series_mean` exists here.
https://github.com/pola-rs/r-polars/blob/c2ef53b16b73bcf25d4af15a44f6fbfb0373374c/R/series__series.R#L670-L672